### DISCUSSION TITLE
Removing highlight/background color from signs

### DISCUSSION THREAD

**Discussion by alt-romes** [w0rp/ale#3665] [Q&A - Ask for help with problems]
Hi! I'm trying to remove the highlight from behind the `--` sign. I've read through the whole documentation and I still can't figure it out. I'm also pretty sure I've tried every possible combination of `highlight clear ALE...` and disabling global variables (of course I haven't :)).
It would be great if someone could help me change the background color of `--` :)
<img width="274" alt="Screenshot 2021-03-29 at 01 19 48" src="https://user-images.githubusercontent.com/21295306/112773202-01058880-902d-11eb-82fe-0e1d9fe25bf8.png">

**Comment by hsanson** [comment]
Check help on g:ale_change_sign_column_color:

> :h g:ale_change_sign_column_color

Chaning the SignColumn or ALESignColumnWithErrors highlight should do.

**Reply by gennaro-tedesco** [reply]
```
highlight clear ALEErrorSign
highlight clear ALEWarningSign
```
is what did it for me (`g:ale_change_sign_column_color` does nothing really).
